app_demo.rb

```diff
@@ -0,0 +1,83 @@
+require './lib/order'
+require './lib/submitorder'
+
+class Takeaway
+
+  attr_accessor :menu1, :order1, :name, :price, :menu_item1, :total, :to_mobile, :from_mobile, :message_body, :message, :account_sid, :auth_token, :submitorder1, :quantity
+
+  def initialize
+    @menu1 = Menu.new Dish
+  end
+
+  def select_option
+    puts 'Takeaway Challenge'
+    puts '=================='
+    puts
+    puts 'Select Option:'
+    puts 'A - Add dish to menu'
+    puts 'B - Display menu so that can make an order'
+    puts 'C - Create an order'
+    puts 'D - Total an order'
+    puts 'E - Submit order'
+    puts 'X - End'
+    option = gets.chomp.downcase
+  end
+
+  def demo
```

Comment:
Metrics/CyclomaticComplexity: Cyclomatic complexity for demo is too high. [13/6]<br>Metrics/AbcSize: Assignment Branch Condition size for demo is too high. [79.01/15] (http://c2.com/cgi/wiki?AbcMetric)<br>Metrics/MethodLength: Method has too many lines. [52/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)<br>Metrics/PerceivedComplexity: Perceived complexity for demo is too high. [9/7]